WarshipGame/Assets/Scripts/Gameplay/Grid/HexData.cs

```diff
@@ -29,18 +32,20 @@ private void Awake()
     /// <exception cref="Exception"></exception>
     public new int GetType()
     {
-        switch (hexType)
+        switch (HexType)
         {
-            case HexType.Default:
+            case HexType.FreeSpace:
                 return 1;
+            case HexType.OilWater :
```

Comment:
remove whitespace next to OilWater

Reply:
shoot forgot, can i change that next time? already closed my editor and i have to reopen the Enum list stuff later anyway. i will mark it as clean-up in my personal ToDo list